### PR TITLE
Add docs for ORMs

### PR DESCRIPTION
**Pull request by SferaDev**

### CODE REVIEW

File: 020-Connect/020-JavaScript/020-prisma.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  where: {
+    name: 'Alice'
+  }
+});
+```
+
+### Limitations
+
+- Xata File Storage is read-only and only shows file metadata. You need to use the SDK to upload and download files.
+- Prisma migrate is not fully supported. You need to use the Xata CLI to manage the database migrations.
```
Comment (tsg) [inline 020-Connect/020-JavaScript/020-prisma.mdx:74]: Added a link to https://xata.io/docs/migrations . Those docs are not fully fleshed out, we probably need to enhance them as well a bit. Also added "yet" for our intentions here :) 

```suggestion
- Prisma migrate is not yet fully supported. You need to use the Xata CLI to manage the [database migrations](/docs/migrations).
```

**Review by tsg — APPROVED** [review]

File: 020-Connect/020-JavaScript/030-drizzle.mdx [hunk 2]
```diff
@@ -46,7 +63,7 @@ const db = drizzle(xata);
 To create a drizzle instance with the TCP client, use the following code:
 
 ```ts
-import { drizzle } from '@xata.io/drizzle/pg';
+import { drizzle } from 'drizzle-orm/pg-core';
```
Comment (SferaDev) [inline 020-Connect/020-JavaScript/030-drizzle.mdx:66]:
```suggestion
import { drizzle } from "drizzle-orm/node-postgres";
```

File: 020-Connect/020-JavaScript/030-drizzle.mdx [hunk 3]
```diff
@@ -58,7 +75,7 @@ const db = drizzle(client);
 For production use cases, you can create a pool of connections with the TCP client:
 
 ```ts
-import { drizzle } from '@xata.io/drizzle/pg';
+import { drizzle } from 'drizzle-orm/pg-core';
```
Comment (SferaDev) [inline 020-Connect/020-JavaScript/030-drizzle.mdx:78]:
```suggestion
import { drizzle } from 'drizzle-orm/node-postgres';
```

File: 020-Connect/020-JavaScript/030-drizzle.mdx [hunk 4]
```diff
@@ -74,8 +91,7 @@ const db = drizzle(pool);
 A full example of using Drizzle with Xata is shown below:
 
 ```ts
-import { pgTable, text } from 'drizzle-orm/pg-core';
-import { drizzle } from '@xata.io/drizzle';
+import { drizzle, pgTable, text } from 'drizzle-orm/pg-core';
```
Comment (SferaDev) [inline 020-Connect/020-JavaScript/030-drizzle.mdx:94]:
```suggestion
import { drizzle } from 'drizzle-orm/node-postgres';
import { pgTable, text } from 'drizzle-orm/pg-core';
```

File: 020-Connect/020-JavaScript/020-prisma.mdx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+keywords: ['orm', 'prisma', 'typescript', 'query builder']
+description: Use the Prisma integration as a TypeScript ORM
+slug: integrations/prisma
+published: true
+status: beta
+---
+
+[Prisma ORM](https://www.prisma.io/orm) is a TypeScript ORM that is focused on type safety and developer experience.
+
+### Installation
```
Comment (snide) [inline 020-Connect/020-JavaScript/020-prisma.mdx:13]:
```suggestion
## Installation
```

File: 020-Connect/020-JavaScript/020-prisma.mdx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+const prisma = new PrismaClient({});
+
+const result = await prisma.users.findMany({
+  where: {
+    name: 'Alice'
+  }
+});
+```
+
+### Limitations
```
Comment (snide) [inline 020-Connect/020-JavaScript/020-prisma.mdx:71]:
```suggestion
## Limitations
```

File: 020-Connect/020-JavaScript/020-prisma.mdx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+```sh
+yarn add prisma
+```
+
+</TabbedCode>
+
+### Usage
+
+You can use Prisma with Xata with the default postgres client. To do so, you need to define the schema in the `schema.prisma` file. Here is an example of a schema:
```
Comment (snide) [inline 020-Connect/020-JavaScript/020-prisma.mdx:34]:
```suggestion
You can use Prisma if you have enabled your Xata database to allow [Postgres connections](/docs/postgres). Once enabled, define a schema in the `schema.prisma` file. Here is an example of a schema:
```

**Review by snide — COMMENTED** [review]
I'd suggest adding the same link / qualifier about needing to enable Postgres. Without that context people might not understand why their schema file is not working.

If you want, you can use `<DocsEnablePostgres />` to give instructions on how to enable the DB.

File: 020-Connect/020-JavaScript/030-drizzle.mdx [hunk 8]
```diff
@@ -12,19 +12,36 @@ status: beta
...
 
 ```sh
-pnpm add drizzle-orm @xata.io/drizzle @xata.io/client pg
+pnpm add drizzle-orm @xata.io/client
 ```
 
 ```sh
-yarn add drizzle-orm @xata.io/drizzle @xata.io/client pg
+yarn add drizzle-orm @xata.io/client
+```
```
Comment (snide) [inline 020-Connect/020-JavaScript/030-drizzle.mdx:28]: Don't you need the `next` version for this stuff?

File: 020-Connect/020-JavaScript/030-drizzle.mdx [hunk 9]
```diff
@@ -12,19 +12,36 @@ status: beta
 
 ## Installation
 
-Install both Drizzle and `@xata.io/drizzle` with the Xata client. You can use the following package managers:
+Install Drizzle with the Xata client. You can use the following package managers:
 
 <TabbedCode tabs={['npm', 'pnpm', 'yarn']}>
 ```sh
-npm install drizzle-orm @xata.io/drizzle @xata.io/client pg
+npm install drizzle-orm @xata.io/client
```
Comment (SferaDev) [inline 020-Connect/020-JavaScript/030-drizzle.mdx:19]:
```suggestion
npm install drizzle-orm @xata.io/client@next
```

File: 020-Connect/020-JavaScript/030-drizzle.mdx [hunk 10]
```diff
@@ -12,19 +12,36 @@ status: beta
...
 ```
 
 ```sh
-pnpm add drizzle-orm @xata.io/drizzle @xata.io/client pg
+pnpm add drizzle-orm @xata.io/client
 ```
 
 ```sh
-yarn add drizzle-orm @xata.io/drizzle @xata.io/client pg
+yarn add drizzle-orm @xata.io/client
```
Comment (SferaDev) [inline 020-Connect/020-JavaScript/030-drizzle.mdx:27]:
```suggestion
yarn add drizzle-orm @xata.io/client@next
```

**Comment by alexfrancoeur** [comment]
bubbling this up so we have prisma / drizzle docs when we release. looks like the preview build is broken.

**Review by snide — APPROVED** [review]
Fixed your build and cleaned up the docs to provide more clear installation instructions. Also added links to the referenced materials.

**Pull request merged**